start-client/src/components/common/layout/QuickLinks.js

```diff
@@ -119,6 +119,21 @@ class QuickLinks extends React.Component {
                       What's New With Spring
                     </a>
                   </li>
+
+                  <li>
+                    <a
+                      id='ql-help-comminty'
+                      target='_blank'
+                      rel='noopener noreferrer'
+                      tabIndex='-1'
+                      href='https://github.com/spring-projects/spring-boot/blob/b0626bca320c21928e9790fcd2c59337344f9ac5/spring-boot-project/spring-boot-starters/README.adoc'
```

Comment:
This page is actually different depending on the Spring Boot version you are using. The link between this link and the Spring Boot selector is nonexistent so there's not much we can do about it.

That said, linking to a particular revision is not what I would do. I'd like to get more feedback.